go-runtime/internal/impl.go

```diff
@@ -41,3 +41,20 @@ func (r *RealFTL) FSMSend(ctx context.Context, fsm, instance string, event any)
 	}
 	return nil
 }
+
+func (r *RealFTL) PublishEvent(ctx context.Context, topic string, event any) error {
+	fmt.Printf("impl: publish event")
+	client := rpc.ClientFromContext[ftlv1connect.VerbServiceClient](ctx)
+	body, err := encoding.Marshal(event)
+	if err != nil {
+		return fmt.Errorf("failed to marshal event: %w", err)
+	}
+	_, err = client.PublishEvent(ctx, connect.NewRequest(&ftlv1.PublishEventRequest{
+		Topic: &schemapb.Ref{Module: reflection.Module(), Name: topic},
```

Comment:
What if you're publishing to a module that's from another module?

Reply:
Good point, I've made it an issue and part of the pubsub epic
https://github.com/TBD54566975/ftl/issues/1640